### PR TITLE
Feature lwdev 2929 accounts transactions reports

### PR DESCRIPTION
**Pull request by mihail-yartsev**

### CODE REVIEW

File: src/MarginTrading.Brokers/MarginTrading.AccountHistoryBroker/Application.cs [hunk 1]
```diff
@@ -1,75 +1,64 @@
...
+                Amount = (double) accountHistoryContract.Amount,
+                Balance = (double) accountHistoryContract.Balance,
+                Date = accountHistoryContract.Date,
+                Type = accountHistoryContract.Type.ToString(),
+                WithdrawTransferLimit = (double) accountHistoryContract.WithdrawTransferLimit,
+            };
+            _accountTransactionsReportsRepository.InsertOrReplaceAsync(accountTransactionReport);
+            return Task.WhenAll(
+                _accountHistoryRepository.AddAsync(accountHistory),
+                _accountTransactionsReportsRepository.InsertOrReplaceAsync(accountTransactionReport));
```
Comment (andrey-anokhin) [inline src/MarginTrading.Brokers/MarginTrading.AccountHistoryBroker/Application.cs]: Duplicated report task?

File: src/MarginTrading.Brokers/MarginTrading.AccountMarginEventsBroker/Application.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            _accountMarginEventsReportsRepository = accountMarginEventsReportsRepository;
+        }
+
+        protected override RabbitMqSubscriptionSettings GetRabbitMqSubscriptionSettings()
+        {
+            var exchangeName = _settings.RabbitMqQueues.AccountMarginEvents.ExchangeName;
+            return new RabbitMqSubscriptionSettings
+            {
+                ConnectionString = _settings.MtRabbitMqConnString,
+                QueueName = $"{exchangeName}.{_applicationInfo.ApplicationName}.{_settings.Env ?? "DefaultEnv"}",
```
Comment (andrey-anokhin) [inline src/MarginTrading.Brokers/MarginTrading.AccountMarginEventsBroker/Application.cs]: Why not QueueHelper.BuildQueueName?

File: src/MarginTrading.Brokers/MarginTrading.BrokerBase/BrokerStartupBase.cs [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+            logAggregate.AddLogger(logToConsole);
+
+            var slackService = settings.SlackNotifications != null
+                ? services.UseSlackNotificationsSenderViaAzureQueue(settings.SlackNotifications.AzureQueue,
+                    logToConsole)
+                : null;
+
+            // Creating azure storage logger, which logs own messages to concole log
+            var dbLogConnectionString = settings.MtBrokersLogs?.DbConnString;
```
Comment (andrey-anokhin) [inline src/MarginTrading.Brokers/MarginTrading.BrokerBase/BrokerStartupBase.cs]: All Demo and Live logs will be in one table and we could never understand which one is error source? May be at least use isLive property in table names?

File: src/MarginTrading.Services/AccountManager.cs [hunk 4]
```diff
@@ -81,23 +87,69 @@ private Task WriteAccountsStats(IReadOnlyList<MarginTradingAccount> accounts)
             return _statsRepository.InsertOrReplaceBatchAsync(stats);
         }
 
+
+
+        private Task WriteAccountsStatsReports(IReadOnlyList<MarginTradingAccount> accounts)
```
Comment (andrey-anokhin) [inline src/MarginTrading.Services/AccountManager.cs]: WriteAccountsStats creates very similar objects. May be it is better to use calculated values? Also need to think about performance here....

File: src/MarginTrading.Brokers/MarginTrading.BrokerBase/BrokerStartupBase.cs (hunk 3 above)
Comment (mihail-yartsev) [reply src/MarginTrading.Brokers/MarginTrading.BrokerBase/BrokerStartupBase.cs]: The connection strings to databases should be different then

**Pull request merged**